### DISCUSSION TITLE
sitemap is not getting updated

### DISCUSSION THREAD

**Discussion by dmproj** [vercel/next.js#72953] [Help]
### Summary

Sitemap is not updating. I have pushed the updated file to Vercel multiple times and purged the cache, but the changes are still not reflected. The updated file is present in the github. I have 1 root file sitemap.xml and 2 branches - sitemap1 and sitemap2, branches are updating, the root one doesnt'.

**Comment by icyJoseph** [comment]
Link to project/repository?

**Reply by dmproj** [reply]
.

**Reply by icyJoseph** [reply]
I experienced something similar last week. A route handler's GET method, wouldn't update at all.. though it was a dynamic endpoint though ~

I had to update my Next.js version to get a fresh build running, and that seemed to fix it

**Reply by dmproj** [reply]
it's quite strange. the new sitemap appears when I check the source on Vercel, but the outdated file is still served when accessing the sitemap via  browses

**Reply by icyJoseph** [reply]
Do you know how to curl or get the sitemap by other means?
